dotnet/samples/KernelSyntaxExamples/Program.cs

```diff
@@ -20,61 +20,61 @@ public static async Task Main()
+        // await Example20_HuggingFace.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example21_ChatGptPlugins.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example22_OpenApiSkill_AzureKeyVault.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example23_OpenApiSkill_GitHub.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example24_OpenApiSkill_Jira.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example25_ReadOnlyMemoryStore.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example26_AADAuth.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example27_SemanticFunctionsUsingChatGPT.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example28_ActionPlanner.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example29_Tokenizer.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example30_ChatWithPrompts.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example31_CustomPlanner.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example32_StreamingCompletion.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example33_StreamingChat.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example34_CustomChatModel.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example35_GrpcSkills.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example36_MultiCompletion.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example37_MultiStreamingCompletion.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example38_Pinecone.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example39_Postgres.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example40_DIContainer.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example41_HttpClientUsage.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example42_KernelBuilder.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example43_GetModelResult.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example44_MultiChatCompletion.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example45_MultiStreamingChatCompletion.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example46_Weaviate.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example47_Redis.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example48_GroundednessChecks.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example49_LogitBias.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example50_Chroma.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example51_StepwisePlanner.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example52_ApimAuth.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example53_Kusto.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example54_AzureChatCompletionWithData.RunAsync().SafeWaitAsync(cancelToken);
+        // await Example55_TextChunker.RunAsync().SafeWaitAsync(cancelToken);
```

Comment:
Did you mean to include this commenting?

Reply:
Nope, resolved.